### DISCUSSION TITLE
mapping keyboard key to mouse button only in certain app(s)

### DISCUSSION THREAD

**Discussion by photopoeth** [pqrs-org/Karabiner-Elements#4037] [Q&A]
As a recent „Win2Mac switcher“ I am used to use a conventional mouse in the browser to click links with the 3rd button to open them in a new tab.  My new Magic Mouse does not have a third button, so I have mapped right_option to mouse->button3 via “Simple Modifications”. This works perfect. But…
Now I want to limit this mapping exclusively to the browser(s), which is obviously not possible with “Simple Modification”. So I want to create a “Complex Modification” for this. But there I have no access to the mouse buttons, right?
Is there a solution for this?

—-BR, Thomas

**Comment by tekezo** [comment]
This json changes right option to button3 only on Safari.

```json
{
    "description": "Change right_option to button3 on Safari",
    "manipulators": [
        {
            "conditions": [
                {
                    "bundle_identifiers": [
                        "^com\\.apple\\.Safari$"
                    ],
                    "type": "frontmost_application_if"
                }
            ],
            "from": {
                "key_code": "right_option",
                "modifiers": { "optional": ["any"] }
            },
            "to": [{ "pointing_button": "button3" }],
            "type": "basic"
        }
    ]
}
```

**Reply by photopoeth** [reply]
Oh, I see... "pointing-button" is doing the trick. I overlooked that. Thanks a lot!
